src/foam/core/AbstractPropertyInfo.java

```diff
@@ -6,8 +6,7 @@
 
 package foam.core;
 
-import foam.dao.pg.IndexedPreparedStatement;
-import foam.lib.xml.Outputter;
+import foam.dao.jdbc.IndexedPreparedStatement;
```

Comment:
Why is this needed?

Reply:
It is not, I made a refactoring to put JDBC classes into foam.dao.jdbc instead of foam.dao.pg. I'll remove it

Reply:
It is used by the prepareStatement method, I only made a refactoring of the packages